source/about.html.slim

```diff
@@ -0,0 +1,24 @@
+---
+title: About
+---
+
+section
+  .body-inner-wrapper
+    .body-inner-wrapper__section#about
+      h2 📕 About Kiwi Ruby
+      p Kiwi Ruby is the first Ruby conference in New Zealand.
+      p Ruby is a friendly and interesting language, and the community is full of friendly and interesting people.
+      p We want to give the community the opportunity to learn and grow, and we want to showcase the talent we have right here in New Zealand.
+    .body-inner-wrapper__section#signup
+      h2 ✨ Diversity, inclusivity, and accessibility
+      p
+        ' We are committed to making Kiwi Ruby accessible and inclusive. We believe it's important that the community is diverse and can accommodate many different people.
```

Comment:
can you read over this copy @raquelxmoss ? Do you think we need a more detailed diversity/accessibility policy/statement?